### PR TITLE
fix: remove JS

### PR DESCRIPTION
**Pull request by craigbilner**
Web images should work without JS enabled which also makes SSR pages more elegant

n.b. still uses JS for placeholder

### CODE REVIEW

File: packages/image/image.web.js [hunk 1]
```diff
@@ -1,62 +1,38 @@
...
-}
+      right: 0,
+      zIndex: 0
+    },
+    img: { display: "block", width: "100%", zIndex: 1, position: "absolute" }
+  };
+
+  const boundedImg = (
+    <div style={styles.wrapper}>
+      <img src={uri} style={styles.img} alt="" />
```
Comment (jonnywyatt) [inline packages/image/image.web.js]: Could alt be passed as a prop?

File: packages/image/image.web.js [hunk 2]
```diff
@@ -1,62 +1,38 @@
...
+    wrapper: {
       height: 0,
       overflow: "hidden",
       paddingBottom: `${100 / aspectRatio}%`
-    };
-
-    const placeholderStyle = {
+    },
+    placeholder: {
       flex: 1,
```
Comment (jonnywyatt) [inline packages/image/image.web.js]: Placeholder component already sets this property

File: packages/image/image.web.js [hunk 3]
```diff
@@ -1,62 +1,38 @@
...
-
-    // divs cannot be styled with the output of Stylesheet.create()
-    // only react native Views accept those ids
-    return style ? <View style={style}>{boundedImg}</View> : boundedImg;
-  }
-}
+      right: 0,
+      zIndex: 0
+    },
+    img: { display: "block", width: "100%", zIndex: 1, position: "absolute" }
```
Comment (jonnywyatt) [inline packages/image/image.web.js]: Does height need 100%?

File: packages/image/image.web.js (hunk 1 above)
Comment (craigbilner) [reply packages/image/image.web.js]: It should be! Will do as a separate PR as need to agree the data too

File: packages/image/image.web.js (hunk 2 above)
Comment (craigbilner) [reply packages/image/image.web.js]: 👍

**Pull request merged**